### PR TITLE
Remove needs-triage label if it is assigned

### PR DESCRIPTION
**Pull request by xiangyan99**
# Description

Please add an informative description that covers that changes made by the pull request and link all relevant issues.

If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.

# All SDK Contribution checklist:
- [ ] **The pull request does not introduce [breaking changes]**
- [ ] **CHANGELOG is updated for new features, bug fixes or other significant changes.**
- [ ] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-python/blob/main/CONTRIBUTING.md).**

## General Guidelines and Best Practices
- [ ] Title of the pull request is clear and informative.
- [ ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-python/blob/main/CONTRIBUTING.md##building-and-testing)
- [ ] Pull request includes test coverage for the included changes.

### CODE REVIEW

File: .github/workflows/remove-needs-triage.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+name: Remove needs-triage label if it is assigned
+
+on:
+  issues:
+    types: [assigned]
```
Comment (jsquire) [inline .github/workflows/remove-needs-triage.yml:5]: The core set of rules also remove this if another label (not "needs-triage") is applied

**Review by jsquire — APPROVED** [review]

**Comment by xiangyan99** [comment]
> .... are none of the Fabric Bot rules working in the Python repository currently?

Most of them are still working.

This is for a bug reported on https://github.com/Azure/azure-sdk-for-python/issues/25300.

Needs-triage label is still added even it is assigned.

File: .github/workflows/remove-needs-triage.yml (hunk 1 above)
Comment (jsquire) [reply .github/workflows/remove-needs-triage.yml:5]: ... and skip it if any labels are already on the issue (including "needs-triage")

Comment (xiangyan99) [reply .github/workflows/remove-needs-triage.yml:5]: I am still investigating how to check "if it is labeled" in action. :)

**Pull request merged**